main.go

```diff
@@ -37,13 +37,15 @@ var flagSet = flag.NewFlagSet("garble", flag.ContinueOnError)
 
 var (
 	flagGarbleLiterals bool
+	flagGarbleTiny     bool
 	flagDebugDir       string
 	flagSeed           string
 )
 
 func init() {
 	flagSet.Usage = usage
 	flagSet.BoolVar(&flagGarbleLiterals, "literals", false, "Encrypt all literals with AES, currently only literal strings are supported")
+	flagSet.BoolVar(&flagGarbleTiny, "tiny", false, "Encrypt all literals with AES, currently only literal strings are supported")
```

Comment:
Documentation string?

Reply:
Probably out of scope for this PR, but the doc string for '-literals' is outdated